Park/Host/Program.cs

```diff
@@ -0,0 +1,195 @@
+                              "И многое другое в нашей новой игре под названием:\n" +
+                              "\"Почему же так сложно, я ведь хотел просто убирать за мартышками!\"\n" +
+                              "                                                   Все права не защищены\n");
+
+            var animalList = new List<Animal>();
+            var plantList = new Plant[3];
+            Initialization(animalList, plantList);
+            int move = 1;
+            Console.WriteLine($"Ход номер: {move}\n");
+            Menu(animalList, plantList, move);
+        }
+
+        static void Initialization(List<Animal> l, Plant[] p)
+        {
+            p[0] = new Grass("Травушка-муравушка");
+            p[1] = new Tree("Деревце кудрявое");
+            p[2] = new Feed("КОМБИКОРМ");
+            l.Add(new AnimalMale("Пушок", null, null, p));
+            l.Add(new AnimalMale("Кренделек", null, null, p));
+            l.Add(new AnimalFemale("Снежинка", null, null, p));
+            l.Add(new AnimalFemale("Звездочка", null, null, p));
+        }
+
+        static void Menu(List<Animal> l, Plant[] p, int move)
+        {
+            Console.WriteLine("Выберите действие: \n" +
+                              "1. Посмотреть список животных и растений.\n" +
+                              "2. Накормить животное.\n" +
+                              "3. Заняться разведением.\n" +
+                              "4. Закончить ход.\n" +
+                              "5. Закрыть игру\n");
+            while (true)
+            {
+                Console.WriteLine("Введите число: ");
+                string choice = Console.ReadLine();
+                int.TryParse(choice, out int x);
+                if (x > 0 && x < 6)
+                {
+                    switch (x)
+                    {
+                        case 1:
+                            GetFullList(l, p, move);
+                            break;
+                        case 2:
+                            GiveFood(l, p, move);
+                            break;
+                        case 3:
+                            MakePregnant(l, p, move);
+                            break;
+                        case 4:
+                            EndMove(l, p, move);
+                            break;
+                        case 5:
+                            Console.WriteLine("Будьте уверены, как только вы покинете игру, все ваши питомцы умрут с голоду. Потому что никому, кроме вас, они нужны.");
+                            Console.WriteLine("Введите Y, чтобы выйти.");
+                            string? exit = Console.ReadLine();
+                            switch (exit)
+                            {
+                                case "Y":
+                                    return;
+                                default:
+                                    Menu(l, p, move);
+                                    break;
```

Comment:
рекурсия)
чтобы было меньше шансов при изменении программы ниже что-то упустить:
```suggestion
                                    Menu(l, p, move);
                                    return;
```

а еще тут наверное [goto](https://professorweb.ru/my/csharp/charp_theory/level3/3_17.php) ложится
хотя это тож не лучшая практика использовать goto